app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        # Check if visitor is vaccinated
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"{visitor.get("name", "Visitor")}"
```

Comment:
Code Efficiency: You are using double quotes inside the f-string, which might cause an error because Python will consider the end of the f-string where the first closing double quote is found. It's better to use single quotes inside the f-string when the f-string itself is enclosed in double quotes.